examples/30_extended/fetch_runtimes_tutorial.py

```diff
@@ -88,32 +86,38 @@ def print_compare_runtimes(measures):
         print("Repeat #{}-Fold #{}: {:.4f}".format(repeat, fold, val2))
     print()
 
+################################################################################
 # The remaining entries recorded in `measures` are the runtime records
 # related as:
 # usercpu_time_millis = usercpu_time_millis_training + usercpu_time_millis_testing
 # wall_clock_time_millis = wall_clock_time_millis_training + wall_clock_time_millis_testing
-
+#
 # The timing measures recorded as `*_millis_training` contain the per
 # repeat-fold timing incurred for the executing of the `.fit()` procedure
 # of the model. For `usercpu_time_*` the time recorded using `time.process_time()`
 # is converted to milliseconds and stored. Similarly, `time.time()` is used
-# to record the time entry for `wall_clock_time_*`. *_millis_testing`
+# to record the time entry for `wall_clock_time_*`. The `*_millis_testing` entry
 # follows the same procedure but for time taken for the `.predict()` procedure.
 
 # Comparing the CPU and wall-clock training times of the Random Forest model
 print_compare_runtimes(measures)
 
-############################################################################
-# Case 2: Running a Random Forest model on an OpenML task in parallel
-# *******************************************************************
+################################################################################
+# Case 2: Running a Random Forest model on an OpenML task in parallel (2 cores)
+# *****************************************************************************
 # Redefining the model to allow parallelism with `n_jobs=2`
 clf = RandomForestClassifier(n_estimators=10, n_jobs=2)
 
-run2 = openml.runs.run_model_on_task(model=clf, task=task, dataset_format="array")
+run2 = openml.runs.run_model_on_task(
+    model=clf, task=task, n_jobs=4, upload_flow=False, avoid_duplicate_runs=False,
+)
 measures = run2.fold_evaluations
 # The wall-clock time recorded per fold should be lesser than Case 1 above
 print_compare_runtimes(measures)
 
+####################################################################################
+# Running a Random Forest model on an OpenML task in parallel (all cores available):
+
```

Comment:
This cell tries to upload the flow implicitly - could you please set `upload_flow=False`? I'm wondering whether we should change the overall behavior here? CC @PGijsbers

Reply:
Aah I realize that I am not being consistent with the call signature to `run_model_on_task` in this cell compares to the previous ones. Here, I am not passing `upload_flow` and `avoid_duplicate_runs`. 
The main reason I believe I did that in the previous two cells is to make it explicit and acquaint the user with these parameters, given this is a tutorial.
I'll make the call consistent here.

However, `upload_flow=False` is the default `run_model_on_task`.